### PR TITLE
Popups

### PR DESCRIPTION
**Pull request by eldang**
Tabbed popups are working!  Some notes:

1. The first time you sync this from Github expect it to take a while.  For some reason all the mp3s had been previously truncated at 31 seconds; this PR fixes that but it means there's a lot of data to pull.
2. In theory the a browser which can't handle the HTML5 `audio` tag should just get a link to download the MP3 instead, but I haven't tested this yet.
3. I think I'd better convert the list of photos to a carousel along the lines of https://www.w3schools.com/howto/howto_js_slideshow.asp .  That tutorial looks pretty easy to follow, I just ran out of time today.
4. I tried to put fields in a logical order in the popups and give them sensible header text, but I bet you'll want to rearrange it some and hopefully that's easy to do.
5. I have only done the absolute bare minimum of styling work.
6. I'm trying to work out why Dorothy Starr (6411 Seaview)'s audio link doesn't work.  The file appears to be fine.

There are bound to be other issues that come up as you go, but hopefully this gives you something solid to work with.

### CODE REVIEW

**Comment by eldang** [comment]
@coregis sorry I requested review before writing the comment!  It should make sense if you click through now.

File: scripts/functions.js [hunk 1]
```diff
@@ -610,18 +610,151 @@ function addVectorLayer(map, params) {
...
+		html += '<p>No records found.</p>';
+	} else {
+		html += '<div id="personTabs">';
+		html += '<ul>' + names + '</ul>';
+		html += entries;
+		html += '</div>';
+	}
+
 	return html; //this will return the string to the calling function
 }
```
Comment (eldang) [inline scripts/functions.js:759]: This is where to reorder if fields if needed

File: css/ballard-4culture-style.css [hunk 2]
```diff
@@ -17,11 +17,18 @@ div#map {
 	  bottom:0;
 }
 
-/* This formatting is for straight GL JS and may be superseded by EG's super cool fillpopup_ function down the road */
```
Comment (coregis) [inline css/ballard-4culture-style.css:20]: Ha ha I cannot wait to see the **_super cool_** fillpopup_ function!!

File: scripts/functions.js (hunk 1 above)
Comment (coregis) [reply scripts/functions.js]: Thank you for calling this out, that's very helpful!

**Review by coregis — APPROVED** [review]
This all seems to make sense upon cursory review, I'm going to merge this into main, then create a new branch for styling pop-ups and making a handful of other content-related changes requested by BHS.

**Pull request merged**